### PR TITLE
Update Search URL to include parameterized search

### PR DESCRIPTION
**Pull request by manashar4**
This update adds the documentation around existing capability of Search URL to support Parameterized Search

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/search/get-started-with-search/build-search/use-url-to-run-search.md [hunk 1]
```diff
@@ -38,3 +41,22 @@ The custom URL that launches this log query in the Sumo Logic Search page woul
...
+{{app_name}} error | count
+```
+
+Assuming that user selects `billing` app from the list of apps, the final query is : 
+
+```sql
+billing error | count
+```
+
+The custom URL that launches this log query in the Sumo Logic Search page would be similar to the following. The exact URL would depend on your Sumo Logic account endpoint, as listed in [Sumo Logic Endpoints and Firewall Security](/docs/api/getting-started#sumo-logic-endpoints-by-deployment-and-firewall-security).
```
Comment (jpipkin1) [inline docs/search/get-started-with-search/build-search/use-url-to-run-search.md:58]:
```suggestion
The custom URL that launches this log query in the Sumo Logic Search page would be similar to the following. The exact URL would depend on your Sumo Logic account endpoint, as listed in [Sumo Logic Endpoints by Deployment and Firewall Security](/docs/api/getting-started#sumo-logic-endpoints-by-deployment-and-firewall-security). 
```

File: docs/search/get-started-with-search/build-search/use-url-to-run-search.md [hunk 2]
```diff
@@ -38,3 +41,22 @@ The custom URL that launches this log query in the Sumo Logic Search page woul
 ```
 https://service.sumologic.com/ui/#/search/create?query=test%20query%20%7C%20count%20by%20_source&startTime=-10w&endTime=-1w
 ```
+### Templatized Query URL
```
Comment (jpipkin1) [inline docs/search/get-started-with-search/build-search/use-url-to-run-search.md:44]:
```suggestion
### Templatized query URL
```

File: docs/search/get-started-with-search/build-search/use-url-to-run-search.md [hunk 3]
```diff
@@ -25,7 +26,9 @@ https://<endpoint>/ui/#/search/create?query=<urlEncodedQuery>&startTime=<start>&
     * in milliseconds since epoch.
     * a [relative time range expression](../search-basics/time-range-expressions.md).  You can omit the `<end>` value and the current time (`now`) is assumed.
 
-## Example
+## Examples
+
+### Standard Query URL
```
Comment (jpipkin1) [inline docs/search/get-started-with-search/build-search/use-url-to-run-search.md:31]:
```suggestion
### Standard query URL
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
This PR looks great, @manashar4. I had a few minor suggestions. If you like them, click the "Commit suggestion" button. After you have resolved all the comments, I will approve the PR.

**Comment by jpipkin1** [comment]
@manashar4, also click the "sign your name here" link in the **cla-bot** box above and follow the directions.

**Review by kimsauce — APPROVED** [review]

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**